homeassistant/components/airly/config_flow.py

```diff
@@ -110,6 +110,4 @@ async def test_location(
 
     current = measurements.current
 
-    if current["indexes"][0]["description"] == NO_AIRLY_SENSORS:
-        return False
-    return True
+    return bool(current["indexes"][0]["description"] != NO_AIRLY_SENSORS)
```

Comment:
Why wrap it with `bool`?
```suggestion
    return current["indexes"][0]["description"] != NO_AIRLY_SENSORS
```

Reply:
This is actually `mypy` that needs/wants this. As an equality could be implemented / overridden that doesn't return a boolean... 🙄

Reply:
> This is actually `mypy` that needs/wants this. As an equality could be implemented / overridden that doesn't return a boolean... 🙄

It's actually both mypy and pyright. Since `current["indexes"][0]["description"]` is `Any`, the result would also be `Any` / `Unknown`. That's how it is 🤷🏻‍♂️

Reply:
Interestingly, my pyright doesn't complain about it, but mypy does.